source/NVDAObjects/__init__.py

```diff
@@ -974,6 +975,9 @@ def event_becomeNavigatorObject(self, isFocus=False):
 		@param isFocus: true if the navigator object was set due to a focus change.
 		@type isFocus: bool
 		"""
+		# #2559: coordinate tone playback takes priority.
+		if config.conf["presentation"]["playObjectCoordinates"]:

```

Comment:
Hi,

event_becomeNavigatorObject was chosen as it can cover a whole bunch of scenarios. The original idea was to modify global commands, but I and others realized that this will not scalewell, hence this method.

Thanks.